MobileSyncExplorerReactNative/ios/Podfile

```diff
@@ -1,3 +1,6 @@
+require_relative '../node_modules/react-native/scripts/react_native_pods'
```

Comment:
We should have a similar script in our iOS repo to simplify the consumption of the SDK in all pod files.